bin/codegen-utils/boilerplate.go.txt

```diff
@@ -0,0 +1,16 @@
+/*
```

Comment:
why is this needed?

Reply:
we talked about this - it's so that the generated code will use our own boilerplate